test/system/065-cp.bats

```diff
@@ -949,9 +949,40 @@ ${randomcontent[1]}" "$description"
     run_podman rm -t 0 -f cpcontainer
 }
 
+@test "podman cp --overwrite file" {
+    rand_content_file=$(random_string 50)
+    rand_content_dir=$(random_string 50)
+
+    run_podman run -d --name ctr-file $IMAGE sleep infinity
+    run_podman exec ctr-file sh -c "echo '$rand_content_file' > /tmp/foo"
+
+    run_podman run -d --name ctr-dir $IMAGE sleep infinity
+    run_podman exec ctr-dir sh -c "mkdir /tmp/foo; echo '$rand_content_dir' > /tmp/foo/file.txt"
```

Comment:
same here
Also is there a reason you test between two containers instead of container to host?